### PR TITLE
feat: Using devfile.io samples for getting-started

### PR DESCRIPTION
**Pull request by ibuziuk**
Screenshot 2024-06-11 at 18 06 28

### CODE REVIEW

File: src/lib/try/TryOnline.svelte [hunk 1]
```diff
@@ -37,52 +37,52 @@
...
         <ReactJs size={70} color={$darkModeThemeEnabled ? 'currentColor' : "#61DAFB"}/>
     </TryTechnology>
 
     <TryTechnology hrefLink='https://workspaces.openshift.com#https://registry.devfile.io/devfiles/dotnet60/1.0.3&storageType=ephemeral'>
         <DotNet size={70}  color={$darkModeThemeEnabled ? 'currentColor' : "#512BD4"}/>
     </TryTechnology>
 
-    <TryTechnology hrefLink='https://workspaces.openshift.com/f?url=https://github.com/che-samples/helloworld-rust&storageType=ephemeral'>
-        <Rust size={70} color={$darkModeThemeEnabled ? 'currentColor' : "#000000"}/>
+    <TryTechnology hrefLink='https://workspaces.openshift.com#https://registry.devfile.io/devfiles/nodejs-basic&storageType=ephemeral'>
```
Comment (l0rd) [inline src/lib/try/TryOnline.svelte:60]:
```suggestion
    <TryTechnology hrefLink='https://workspaces.openshift.com#https://registry.devfile.io/devfiles/python-django&storageType=ephemeral'>
```

**Review by l0rd — COMMENTED** [review]
I have suggested a correction for the Django sample that points to a nodejs-basic.
Apart from that there are a couple of samples that don't use the devfile registry yet (but that may be completely normal and it's ok):

- Quarkus uses the che-incubator sample
- Ollama uses redhat-developer-demos sample

**Comment by ibuziuk** [comment]
@l0rd thanks for the review
updated Ollama sample with https://registry.devfile.io/devfiles/ollama/1.0.0
Regarding Quarkus I believe we would still need to use our sample from the `che-incubator` due to https://github.com/eclipse-che/che/issues/22906

**Comment by l0rd** [comment]
@ibuziuk I get an error when I try to start the Ollama sample:
image

The URL I have used: `https://workspaces.openshift.com#https://registry.devfile.io/devfiles/ollama/1.0.0&storageType=ephemeral`

**Comment by ibuziuk** [comment]
@l0rd I believe it is a resource issue - the new node was not yet provisioned. I think we might need to tune autoscaler setup on Developer Sandbox (we are in the process of migrating production to Argo CD and probably need to do some fine-tuning for DWOC) cc: @dkwon17 there is a separate discussion for skipping FailedScheduling events by default - https://issues.redhat.com/browse/CRW-6590

**Comment by l0rd** [comment]
I wasn't looking for the root cause. If users get a systematic workspace startup error, whatever the cause is, we should remove the sample until that get fixed no?

**Comment by ibuziuk** [comment]
The issue has been reported to the Sandbox folks and for me the sample works now.
The root cause with the situation related to autoscale is going to be fixed as part of https://issues.redhat.com/browse/SANDBOX-659

**Comment by l0rd** [comment]
> The issue has been reported to the Sandbox folks and for me the sample works now. The root cause with the situation related to autoscale is going to be fixed as part of https://issues.redhat.com/browse/SANDBOX-659

I can still reproduce the problem. You think that's because my workspaces get scheduled on a different cluster than yours?

**Review by l0rd — APPROVED** [review]

**Comment by ibuziuk** [comment]
@l0rd could you please comment about this situation in https://issues.redhat.com/browse/SANDBOX-659 
I believe there is some problem with autoscaler, but on m3 it seems to work fine for me right now. 
Thank you for the feedback I will cross-check everything regarding Ollama during contribution of the Developer Sandbox activity - https://issues.redhat.com/browse/SANDBOX-621

**Pull request merged**